docs/site/Testing-your-application.md

```diff
@@ -719,11 +719,8 @@ of the service proxy by invoking the provider. This helper should be typically
 invoked once before the integration test suite begins.
 
 ```ts
-import {
-  GeoService,
-  GeoServiceProvider,
-} from '../../src/services/geo.service.ts';
-import {GeoDataSource} from '../../src/datasources/geo.datasource.ts';
+import {GeoService, GeoServiceProvider} from '../../services/geo.service.ts';
+import {GeoDataSource} from '../../datasources/geo.datasource.ts';
```

Comment:
While you are making these changes: we exclude `.ts` extension in other places, can you remove it from these lines too please?

```suggestion
import {GeoDataSource} from '../../datasources/geo.datasource';
```